test/regression_tests.py

```diff
@@ -1,28 +1,23 @@
-import sys
 import os
-from pygit2 import Repository
+import sys
 
 # To help discover local modules
 REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
 sys.path.append(REPO_ROOT)
 
+import datetime
+
+from ts_scripts import marsgen as mg
+from ts_scripts.api_utils import test_api
 from ts_scripts.install_from_src import install_from_src
 from ts_scripts.regression_utils import test_regression
-from ts_scripts.api_utils import test_api
-from ts_scripts import print_env_info as build_hdr_printer
 from ts_scripts.utils import check_python_version
-from ts_scripts import marsgen as mg
-
-import datetime
 
 now = datetime.datetime.now()
 print("Current date and time : " + now.strftime("%Y-%m-%d %H:%M:%S"))
 
 check_python_version()
 
-git_branch = Repository('.').head.shorthand
```

Comment:
Whats the intention behind this change?

Reply:
This is now working inside the docker container. Also, I'm not sure if this is adding any value. We should remove it.